### DISCUSSION TITLE
如何实现xmake::hidapi依赖包链接到xmake::libusb依赖包

### DISCUSSION THREAD

**Discussion by zkqsky** [xmake-io/xmake#5356] [Q&A] [answered]
add_requires("xmake::libusb")
add_requires("xmake::hidapi")
如果我想让xmake::hidapi能链接到xmake::libusb而不是系统自带libusb，应该怎么写add_requires

**Comment by waruqi** [comment, marked as answer]
add_requires("hidapi")
add_requireconfs("hidapi.libusb", {system = false})
